spyder/plugins/ipythonconsole/widgets/client.py

```diff
@@ -144,7 +144,7 @@ def __init__(self, plugin, id_,
         # To keep a reference to the page to be displayed
         # in infowidget
         self.info_page = None
-        self._show_loading_page()
+        self.before_prompt_is_ready()

```

Comment:
I don't think this needs to be exposed publicly, so please change its name to `_before_prompt_is_ready`